roles/internal/crayfish/defaults/main.yml

```diff
@@ -0,0 +1,73 @@
+crayfish_user: www-data
```

Comment:
since we are in a global namespace, all the variables in this role should be prefixed with the name of the role. 

crayfish_syn_token, etc

Reply:
Will do.

Reply:
Would you include the service-specific ones here? It seems more clear to leave them alone if we expect to break them out into independent roles...

Reply:
Its definitely best practice to have them all namespaces to the role, so there are no conflicts in the global namespace. I'd like to see them all prefixed with the role name.